examples/gno.land/r/gov/dao/v3/README.md

```diff
@@ -0,0 +1,185 @@
+# GOVDAO Specifications
+
+## 1. Overview
+**GovDAO** is a governance body with three membership tiers—**T1**, **T2**, and **T3**—along with defined voting power, membership requirements, size targets, and compensation policies. 
+
+---
+
+## 2. Membership Tiers
+
+### 2.1 Tier Definitions
+- [X] **T1 (Core Tier)**  
+  - [X] Highest tier; self-selecting membership with a *supermajority* vote from T1.
+  - [X] Membership can only be withdrawn by supermajority vote *with cause*.
```

Comment:
`withdrawn by supermajority vote` from T1?

Reply:
Sorry, my comment was really unclear. Actually, I see that to select a new T1 member, the first point in the bullet list specifically mentions that a supermajority of the T1 group is required. But for the withdrawal, the second point in the bullet list just mentions a supermajority (implicitly of all the groups). So I'm wondering if it's error.